src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -440,16 +449,16 @@ void HRVOAgent::computePreferredVelocity()
     if (prefSpeed_ <= 0.01f || max_accel_ <= 0.01f)
     {
         // Used to avoid edge cases with division by zero
-        pref_velocity_ = Vector2(0.f, 0.f);
+        pref_velocity_ = Vector(0.f, 0.f);
         return;
     }
 
     std::unique_ptr<Goal> &nextGoal = simulator_->goals_[goal_index_];
-    Vector2 goalPosition            = nextGoal->getCurrentGoalPosition();
+    Vector goalPosition             = nextGoal->getCurrentGoalPosition();
     float speedAtGoal               = nextGoal->getDesiredSpeedAtCurrentGoal();
-    Vector2 distVectorToGoal        = goalPosition - position_;
-    auto distToGoal = static_cast<float>(std::sqrt(std::pow(distVectorToGoal.getX(), 2) +
-                                                   std::pow(distVectorToGoal.getY(), 2)));
+    Vector distVectorToGoal         = goalPosition - position_;
+    auto distToGoal                 = static_cast<float>(
+        std::sqrt(std::pow(distVectorToGoal.x(), 2) + std::pow(distVectorToGoal.y(), 2)));
```

Comment:
We should use `.length()` here to calculate the distance to goal.

Reply:
Do you mean `static_cast<float>(distVectorToGoal.length());` ?

Reply:
Yes, exactly.